libs/utils/utils.js

```diff
@@ -603,6 +603,7 @@ function decorateContent(el) {
   const block = document.createElement('div');
   block.className = 'content';
   block.append(...children);
+  block.dataset.block = 'true';
```

Comment:
Where is this used?

Reply:
analytics.js L35.  It helps me add an accurate sequence number to the block.
https://github.com/adobecom/milo/blob/476d83100b3ed1a4e9822bdd891b2aa3c54cb175/libs/martech/analytics.js#L35

Reply:
nit: since the value is always true:
```suggestion
  block.dataset.block = '';
```

and in analytics.js the selector can be changed to `[data-block]`

Reply:
updated to empty string